scripts/mg_net/README.md

```diff
@@ -38,28 +46,30 @@ optional arguments:
                         14. Replication Initiation
                         15. Cellular Division
                           
-  -r <min> <max>        Specify the minimum and maximum value for the "discrete uniform" distribution 
+  `-r <min> <max>`:        Specify the minimum and maximum value for the "discrete uniform" distribution 
                         for the random initialization of species initial count (default from 10 to 1000)
-  -n <mean> <range_value_from_mean>
+
+  `-n <mean> <range_value_from_mean>`:
                         Specify the mean and the range value from mean for the normal distribution 
                         for the initialization of k coefficient used in the reaction rate formulas
                         (default mean=5 and range value from mean=2.5)
 
-Examples:
+### Examples:
 
-1.The following example generates a subnetwork in SBML with the reactions belonging to the "Replication Initiation" and "Cellular Division" cellular processes.
-  
+1. The following example generates a subnetwork in SBML with the reactions belonging to the "Replication Initiation" and "Cellular Division" cellular processes.
+  ```
   python3 generate_network.py -s mg_net.sbml -g mg_net.graphml -p 14 15
-
-2.This example generates a network in SBML with all the reactions of mg_net model but with a specific min and max value for 
+  ```
+2. This example generates a network in SBML with all the reactions of mg_net model but with a specific min and max value for 
   the "discrete uniform" distribution for the random initialization of species initial count and save the output to an output file.
-
+  ```
   python3 generate_network.py -s mg_net.sbml -g mg_net.graphml -r 100 2000 -o output.txt
-
-3.The following example generates a subnetwork in SBML with only the reactions of the "Replication" cellular process where specific mean 
+  ```
+3. The following example generates a subnetwork in SBML with only the reactions of the "Replication" cellular process where specific mean 
```

Comment:
The following example generates a subnetwork in SBML including only the reactions of the "Replication" cellular process while randomly assigning the reaction rate coefficient following a normal distribution with mean m and variance v with the bounding interval between lb and ub.

Reply:
Done